unlock-app/src/components/content/SettingsContent.tsx

```diff
@@ -18,14 +18,16 @@ interface PaymentSettings {
 }
 
 export const PaymentSettings = ({ address }: PaymentSettings) => {
-  const { cards, loading } = useCards(address)
+  const { cards, loading, saveCard } = useCards(address)
   if (loading) {
     return <Loading />
   }
   if (cards.length > 0) {
     return <PaymentMethods cards={cards} />
   }
-  return <PaymentDetails />
+  return (
+    <PaymentDetails saveCard={(token: string) => saveCard(address, token)} />
```

Comment:
Because we pass an address into useCards, do you think it would make sense to do this wrapping internal to the hook?

Reply:
yes, I'll fix this!